Added Card Component and RelatedProducts Block

Mikhail-Golovai opened the pull request:
Added Card component and RelatedProducts block
# Card
![related_5](https://user-images.githubusercontent.com/59365124/75635380-9bc34600-5c26-11ea-8629-4553c77a0b70.png)

# RelatedProducts
## In case of more than 3 elements
![related_4](https://user-images.githubusercontent.com/59365124/75635279-ded0e980-5c25-11ea-82a1-02656368eba0.png)
## Large screen
![related_1](https://user-images.githubusercontent.com/59365124/75635281-e1334380-5c25-11ea-88cc-48b2ea1c6d8c.png)

## Medium screen
![related_2](https://user-images.githubusercontent.com/59365124/75635283-e1cbda00-5c25-11ea-8cad-d035c629033a.png)

## Small screen
![related_3](https://user-images.githubusercontent.com/59365124/75635284-e2647080-5c25-11ea-961a-942000f88189.png)

Mikhail-Golovai committed:
Added Card Component and RelatedProducts Block

Kirrrusha commented on src/Blocks/RelatedProducts/RelatedProducts.scss:
```diff
@@ -0,0 +1,55 @@
+                width: 33.33%;
+                @include md(width, 50%);
+                @include xs(width, 100%);
+                /*
```
если не нужен удаляй

Mikhail-Golovai committed:
Cosmetic fixes due to review notes:
- added hover on card buttons
- changed font
- removed dots from RelatedComponents slider

Kirrrusha committed:
fixes

Kirrrusha committed:
Merge remote-tracking branch 'origin/develop' into feature/related_products

# Conflicts:
#	src/App.js

Kirrrusha committed:
fixes

Mikhail-Golovai committed:
Merge branch 'develop' into feature/related_products

Mikhail-Golovai committed:
Merge branch 'develop' into feature/related_products
